core/src/navigation/ContextSwitcher.html

```diff
@@ -139,6 +161,7 @@ <h2 class="fd-title fd-title--h5" id="dialog-title-3">
   let store = getContext('store');
   let getTranslation = getContext('getTranslation');
   let prevContextSwitcherToggle = false;
```

Comment:
```suggestion
  let prevContextSwitcherToggle = false;
  let selectedNodePath;
```
-Follow-up of previous comment on removing **export**.
...I would remove the **export** and declare the variable here, alongside the other non-exported local variables